src/components/home/Home.js

```diff
@@ -1,29 +1,40 @@
+/* eslint-disable no-unused-vars */
 // eslint-disable-next-line no-unused-vars
-import React, { useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 // eslint-disable-next-line no-unused-vars
 import { useSelector, useDispatch } from 'react-redux';
 // eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Please, consider that when the app is opening, it is not displaying the list of items. It looks like this:
![image](https://user-images.githubusercontent.com/49064106/189503464-b63f294d-c9eb-4b6c-9466-b144b456b18d.png)

Reply:
Yeah the id you were using has already reach its free request limit , now it's ok updated a new one